.github/workflows/benchmarks.yml

```diff
@@ -20,7 +20,7 @@ jobs:
         with:
           artifact: npm-package
           workflow: build-test.yml
-          required: false
+          required: true
       - run: mv preact.tgz preact-main.tgz
```

Comment:
This is very much required, as the very next step (as well as other steps later in the flow) require that asset to exist. Therefore, setting `required: false` just moves the error lower, to the attempt to move the (non-)extracted asset.